dessia_common/exports.py

```diff
@@ -317,6 +317,8 @@ def _value_to_str(self, value: Any) -> str:
             return self._sequence_to_str(value)
         if isinstance(value, Dict):
             return self._dict_to_str(value)
+        if value is None:
+            return '-'
```

Comment:
It looks like the returned value differs from the previous functions. The later had space around the ` - ` as this one doesn't. Is it intended ?

Reply:
I'm not sure it changes something for the final render of the markdown on web app but i surrounded the - with spaces, which probably wont make the difference but gives a more consistent print